libbeat/docs/monitoring/monitoring-beats.asciidoc

```diff
@@ -1,74 +1,28 @@
-//////////////////////////////////////////////////////////////////////////
-//// This content is shared by all Elastic Beats. Make sure you keep the
-//// descriptions here generic enough to work for all Beats that include
-//// this file. When using cross references, make sure that the cross
-//// references resolve correctly for any files that include this one.
-//// Use the appropriate variables defined in the index.asciidoc file to
-//// resolve Beat names: beatname_uc and beatname_lc.
-//// Use the following include to pull this content into a doc file:
-//// include::../../libbeat/docs/monitoring/configuring.asciidoc[]
-//// Make sure this content appears below a level 2 heading.
-//////////////////////////////////////////////////////////////////////////
-
 [role="xpack"]
 [[monitoring]]
 = Monitoring {beatname_uc}
 
 [partintro]
 --
 
-NOTE: The {monitor-features} for {beatname_uc} require {es} {beat_monitoring_version} or later.
-
-The {stack} {monitor-features} enable you to easily monitor {beatname_uc} from {kib}. For more
-information, see
-{stack-ov}/xpack-monitoring.html[Monitoring the {stack}] and
-{kibana-ref}/beats-page.html[Beats Monitoring Metrics].
+You can use the {stack} {monitor-features} to gain insight into the health of
+{beatname_uc} agents running in your environment. 
```

Comment:
In `master` this line is replaced by:

```
ifndef::apm-server[]
{beatname_uc} instances running in your environment. 	{beatname_uc} instances running in your environment. 
endif::[]	endif::[]
ifdef::apm-server[]	ifdef::apm-server[]
{beatname_uc}. 	{beatname_uc}. 
endif::[]	endif::[]
```

Do we need to backport this fragment in this PR as well?

Reply:
I don't think so, but let's ask @bmorelli25. AFAIK, he only takes copies of master, not other branches.

Reply:
Thanks for the ping. That's correct, so this is fine with me.